rts/System/SpringApp.cpp

```diff
@@ -1012,6 +1012,17 @@ bool SpringApp::MainEventHandler(const SDL_Event& event)
 
 					SaveWindowPosAndSize();
 
+					if (globalRendering->numDisplays > 1 && globalRendering->dualScreenMode) {
```

Comment:
We now recalculate geometry when window is moved with more than 1 display, as dual view will still be rendered on a separate screen.